src/utilities/viewer.cpp

```diff
@@ -119,6 +119,9 @@ class TestViewerWindow final : private D3D11Resources {
   bool mSetInputFocus = false;
   size_t mRenderCacheKey = 0;
 
+  RECT mWindowRect {};

```

Comment:
This should be an `std::optional<>` to avoid edge cases like when windows asks it to start minimized